mr-plonky2-circuits/src/query2/revelation/public_inputs.rs

```diff
@@ -44,6 +47,8 @@ impl<const L: usize> Inputs<L> {
         L,
         // Block Header
         OutputHash::LEN,
+        // Padded uint512
+        16,
```

Comment:
Replace it with `2*u256::NUM_LIMBS`?

Reply:
Fixed in commit [e120525](https://github.com/Lagrange-Labs/Euclid-database/pull/13/commits/e120525ee1e20b8dbfc165a1406e36992c39dfbd).